quickwit/quickwit-control-plane/src/ingest/ingest_controller.rs

```diff
@@ -1103,39 +1183,36 @@ pub(crate) struct RebalanceShardsCallback {
     pub rebalance_guard: OwnedMutexGuard<()>,
 }
 
-/// Finds the shard with the highest ingestion rate on the ingester with the most number of open
-/// shards. If multiple shards have the same ingestion rate, the shard with the lowest (oldest)
+/// Finds a shard on the ingester with the highest number of open
```

Comment:
+1 for choossing randomly a shard instead of choosing the one with the highest ingestion rate.